src/libraries/System.IO.Compression/tests/CompressionStreamUnitTests.Gzip.cs

```diff
@@ -275,7 +277,6 @@ public void DerivedStream_ReadWriteSpan_UsesReadWriteArray()
         }
 
 
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/47563")]
         [Fact]
         public void StreamCorruption_IsDetected()
```

Comment:
This test also needed the RemoteExecutor + enable the switch, no? I am a bit surprised that CI didn't fail.

Reply:
Gzip detects the corruption without the strict validation enabled (thanks to its CRC integrity check, I assume). 

zlib not so much, if I remove the `AppContext.SetSwitch("System.IO.Compression.UseStrictValidation", true)`, the test fails.